.circleci/config.yml

```diff
@@ -22,13 +22,26 @@ jobs:
       - run:
           name: Check for Lint
           command: |
-            curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.15.0
+            curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
             golangci-lint run ./...
       - run:
           name: Run Tests
           command: go test -cover -race -timeout 60s ./...
 
-  get_source:
+  lint_markdown:
+    <<: *defaults
+    docker:
+      - image: node:11-slim
+    steps:
+      - checkout
+      - run:
+          name: Install markdownlint
+          command: npm install -g markdownlint-cli
+      - run:
+          name: Check for Lint
+          command: markdownlint .
```

Comment:
Do we not need `.markdownlint.json` any more? If not, then let's delete it. Or is it found by default?

Reply:
It's found by default now that we are setting the working directory to the root of the repo.